src/index.js

```diff
@@ -171,7 +214,7 @@ AFRAME.registerComponent('streetmix-loader', {
           );
         }
 
-        el.setAttribute('data-layer-name', 'Streetmix • ' + streetmixName);
+        el.setAttribute('data-layer-name', 'Streetmix � ' + streetmixName);
```

Comment:
unicode issue?

Reply:
yes, something strange with my code editor. Will fix in next commit too